src/common/sqonUtils.js

```diff
@@ -165,7 +165,7 @@ const termToSqon = ({ field, value }) => ({
   op: 'in',
   content: {
     field: field,
-    value: [value],
+    value: Array.isArray(value) ? value : [value],
```

Comment:
that's absolutely correct, but here's a trick : [value].flat() should work, as well